src/components/posts/AddPostForm.js

```diff
@@ -0,0 +1,58 @@
+import { useRef } from "react";
+import classes from "./AddPostForm.module.css";
+
+function AddPostForm(props) {
+  const authorInputRef = useRef();
+  // const dateInputRef = useRef();
+  const avatarInputRef = useRef();
+  const contentInputRef = useRef();
+
+  function submitHandler(event) {
+    event.preventDefault();
+    const enteredAuthor = authorInputRef.current.value;
+    // const enteredDate = dateInputRef.current.value;
+    const enteredDate = new Date().toISOString().slice(0, 10);
```

Comment:
Nice update to get the current date instead of getting it from a user input

Reply:
Thanks